### PR TITLE
Switch to using RDMHandler as the entry point for RDM responders.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: firmware/src/rdm_handler.c [hunk 1]
```diff
@@ -130,6 +130,14 @@ void RDMHandler_HandleRequest(const RDMHeader *header,
   }
 }
 
+void RDMHandler_GetUID(uint8_t *uid) {
+  if (g_rdm_handler.active_model) {
```
Comment (peternewman) [inline firmware/src/rdm_handler.c]: Also check it's not the NULL model?

File: firmware/src/rdm_handler.h [hunk 2]
```diff
@@ -104,6 +104,14 @@ void RDMHandler_HandleRequest(const RDMHeader *header,
                               const uint8_t *param_data);
 
 /**
+ * @brief Get the UID of the responder.
+ * @param uid A pointer to copy the UID to; should be at least UID_LENGTH.
```
Comment (peternewman) [inline firmware/src/rdm_handler.h]: [out]

File: firmware/src/rdm_handler.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/rdm_handler.c]: NULL model is an int, this is a pointer to the active model.

Comment (peternewman) [inline firmware/src/rdm_handler.c]: Understood

File: firmware/src/rdm_handler.h (hunk 2 above)
Comment (nomis52) [inline firmware/src/rdm_handler.h]: Done

**Comment by peternewman** [comment]
LGTM

**Comment by peternewman** [comment]
LGTM

**Pull request merged**